clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -1473,6 +1477,8 @@ The #dynamorio::drmemtrace::TRACE_MARKER_TYPE_TIMESTAMP and
 #dynamorio::drmemtrace::TRACE_MARKER_TYPE_CPU_ID markers are modified by the dynamic
 scheduler to reflect the new schedule.  The new timestamps maintain relative ordering
 but should not be relied upon to indicate accurate durations between events.
+When analyzing core-sharded-on-disk traces, be sure to pass `-no_core_sharded` when
+using core-sharded-preferring tools to avoid the trace being re-scheduled yet again.
```

Comment:
Changed to:

When analyzing core-sharded-on-disk traces, `-no_core_sharded` must be passed when
using core-sharded-preferring tools to avoid an error from the framework attempting
to re-schedule the already-scheduled trace.